tests/javascript_newsletter_editor/mochaTestHelper.js

```diff
@@ -15,16 +15,17 @@ global.sinon = sinon;
 global.URL = URL;
 
 if (!global.document || !global.window) {
-  global.document = jsdom('<html><head><script></script></head><body></body></html>', {
-    url: 'http://example.com',
-  }, {
-    FetchExternalResources: ['script'],
-    ProcessExternalResources: ['script'],
-    MutationEvents: '2.0',
-    QuerySelector: false,
-  });
-
-  global.window = document.defaultView;
+  global.document = new JSDOM(
+    '<html><head><script></script></head><body></body></html>',
+    {
+      url: 'http://example.com',
+      runScripts: 'dangerously',
+      resources: 'usable',
+      pretendToBeVisual: true,
```

Comment:
I added this to get rid of some warnings saying that doesn't support requestAnimationFrame.